homeassistant/components/flipr/__init__.py

```diff
@@ -68,9 +70,15 @@ def __init__(self, hass, entry):
 
     async def _async_update_data(self):
         """Fetch data from API endpoint."""
-        return await self.hass.async_add_executor_job(
-            self.client.get_pool_measure_latest, self.flipr_id
-        )
+        try:
+            data = await self.hass.async_add_executor_job(
+                self.client.get_pool_measure_latest, self.flipr_id
+            )
+        except (FliprError) as error:
+            _LOGGER.error(error)
```

Comment:
We prefer not logging the error but just include it in a message as a parameter to the `UpdateFailed` exception that is raised below. The coordinator will log the message appropriately without spam.

Reply:
Done via PR : https://github.com/home-assistant/core/pull/67477